utils/kata-manager.sh

```diff
@@ -374,7 +374,9 @@ install_containerd()
 	local version_desc="latest version"
 	[ -n "$requested_version" ] && version_desc="version $requested_version"
 
-	info "Downloading $project release ($version_desc)"
+	local latest_version=$(curl -s "$containerd_releases_url/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
I'd add `-L` "just in case":

```suggestion
	local latest_version=$(curl -sL "$containerd_releases_url/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/')
```